src/components/calcite-button/calcite-button.tsx

```diff
@@ -138,7 +138,7 @@ export class CalciteButton {
     );
 
     return (
-      <Host dir={dir} hasText={this.hasText}>
+      <Host dir={dir} hasContent={this.hasContent}>
```

Comment:
Why not just set this prop to `reflect: true`?

Reply:
I'm confused on why this is being set on the dom node? A css class should be used on an internal node instead of an invalid undeclared prop.

Reply:
Can you add a css class on the `<Tag>`? Otherwise, an invalid prop is being exposed to the user.